e2e/__tests__/__snapshots__/test-todo.test.js.snap

```diff
@@ -0,0 +1,82 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`shows error messages when called with invalid argument 1`] = `
+"FAIL __tests__/todo_non_string.test.js
+  ● Test suite failed to run
+
+    Invalid first argument: () => {}. Todo must be called with a string.
+
+      6 |  */
+      7 |
+    > 8 | it.todo(() => {});
+        |    ^
+      9 |
+
+      at packages/jest-jasmine2/build/jasmine/Env.js:480:15
```

Comment:
can we strip this from stack trace? cc @SimenB

Reply:
@SimenB please teach me this magic (again), I've tried but not been able to remove it :(